pymatviz/utils.py

```diff
@@ -46,7 +46,7 @@
 elements_csv = f"{ROOT}/pymatviz/elements.csv"
 df_ptable = pd.read_csv(elements_csv, comment="#").set_index("symbol")
 
-ELEM_CLASS_COLORS: Final = {
```

Comment:
Thanks! I was not aware its existence here. 

And yes I renamed everything to "element type" instead of "element class" (also used "elem" over "ele" as the abbreviation), because previously these two are used inconsistently across ptable plotters :)